models/DataObject/ClassDefinition/Data/ReverseObjectRelation.php

```diff
@@ -208,6 +244,28 @@ public function isOptimizedAdminLoading(): bool
     {
         return true;
     }
+
+    protected function enrichDataRow($object, array $params, ?string &$classId, &$row = [], string $srcCol = 'src_id')
```

Comment:
Right, I removed it because Reverse relations are set "untouchable" in 
https://github.com/pimcore/pimcore/blob/af69638badd601feb3b4d695e6eec0eb9d7462bb/models/DataObject/Concrete/Dao.php#L205-L211
and thus in
https://github.com/pimcore/pimcore/blob/af69638badd601feb3b4d695e6eec0eb9d7462bb/models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php#L89-L91
saving gets skipped and so https://github.com/pimcore/pimcore/blob/af69638badd601feb3b4d695e6eec0eb9d7462bb/models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php#L117 does never get called.

Removed it (above explanation is only for our future selfs ;-)

Reply:
Right, I removed it because Reverse relations are set "untouchable" in 
https://github.com/pimcore/pimcore/blob/af69638badd601feb3b4d695e6eec0eb9d7462bb/models/DataObject/Concrete/Dao.php#L205-L211
and thus in
https://github.com/pimcore/pimcore/blob/af69638badd601feb3b4d695e6eec0eb9d7462bb/models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php#L89-L91
saving gets skipped and so https://github.com/pimcore/pimcore/blob/af69638badd601feb3b4d695e6eec0eb9d7462bb/models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php#L117 does never get called.

Removed it (above explanation is only for our future selfs ;-)